frame/example/src/lib.rs

```diff
@@ -37,13 +37,13 @@
 //!         Capitalize the first word of each documentation comment and end it with
 //!         a full stop. See
 //!         <a href="https://github.com/paritytech/substrate#72-contributing-to-documentation-for-substrate-packages"
-//!         target="_blank"> Generic example of annotating source code with documentation comments</a></li>
-//!     <li>Self-documenting code - Try to refactor code to be self-documenting.</li>
-//!     <li>Code comments - Supplement complex code with a brief explanation, not every line of code.</li>
-//!     <li>Identifiers - surround by backticks (i.e. <code>INHERENT_IDENTIFIER</code>, <code>InherentType</code>,
-//!         <code>u64</code>)</li>
-//!     <li>Usage scenarios - should be simple doctests. The compiler should ensure they stay valid.</li>
-//!     <li>Extended tutorials - should be moved to external files and refer to.</li>
+//!         target="_blank"> Generic example of annotating source code with documentation
+//! comments</a></li>     <li>Self-documenting code - Try to refactor code to be
+//! self-documenting.</li>     <li>Code comments - Supplement complex code with a brief explanation,
+//! not every line of code.</li>     <li>Identifiers - surround by backticks (i.e.
+//! <code>INHERENT_IDENTIFIER</code>, <code>InherentType</code>,         <code>u64</code>)</li>
+//!     <li>Usage scenarios - should be simple doctests. The compiler should ensure they stay
+//! valid.</li>     <li>Extended tutorials - should be moved to external files and refer to.</li>
 //!     <!-- Original author of paragraph: @AmarRSingh -->
```

Comment:
how does this look in rust doc?

Reply:
I just aignored this doc block, I don't know why we have HTML stuffed in there.

Reply:
Hmmm, this isn't great